src/Components/Admin/EnvironmentEditor/transitionModal.js

```diff
@@ -44,6 +59,14 @@ export default function TransitionModal({
         setTransitions([...reorderedList]);
     };
 
+    const addTransition = () => {
+        const newTransition = {
+            text: prompt("Enter the text for the transition: "),
+        };
+
+        setTransitions([...transitions, newTransition]);
+    };
```

Comment:
Lol we'll definitely be changing this in the future but works for now I guess

Reply:
We need to be careful with popups within popups. Will need to explore a better solution, but I'll wait until designs are complete before using the engineering time.